MapboxNavigation/UIFont.swift

```diff
@@ -37,14 +37,14 @@ extension UIFont {
     
     func with(fontFamily: String?) -> UIFont {
         guard let fontFamily = fontFamily else { return self }
-        let weight = (fontDescriptor.object(forKey: UIFontDescriptorTraitsAttribute) as! [String:Any])[UIFontWeightTrait]
-        let attributes = [UIFontDescriptorTraitsAttribute: [UIFontWeightTrait: weight]]
+        let weight = (fontDescriptor.object(forKey: UIFontDescriptor.AttributeName.traits) as! [String:Any])[UIFontDescriptor.TraitKey.weight.rawValue]
+        let attributes = [UIFontDescriptor.AttributeName.traits: [UIFontDescriptor.TraitKey.weight: weight]]
```

Comment:
Inline this variable so that you can drop `UIFontDescriptor.AttributeName` and possibly `UIFontDescriptor.TraitKey`.